quinn-proto/src/connection/streams/mod.rs

```diff
@@ -362,12 +362,12 @@ impl Ord for PendingLevel {
 /// Application events about streams
 #[derive(Debug, PartialEq, Eq)]
 pub enum StreamEvent {
-    /// One or more new streams has been opened
+    /// One or more new streams has been opened and might be readable.
```

Comment:
nit: we avoid trailing commas on the first line of doc comments
```suggestion
    /// One or more new streams has been opened and might be readable
```